Makefile

```diff
@@ -35,7 +35,7 @@ jenkins-test: jenkins-tools mod-lint build linters license
 	TEST_PACKAGES=github.com/onosproject/device-provisioner/... ./build/build-tools/build/jenkins/make-unit
 
 integration-tests:  # @HELP run helmit integration tests locally
```

Comment:
FYI there is a makefile Ray added you can create namespace test by calling that target. https://github.com/onosproject/onos-topo/blob/master/Makefile#L38